Specs/DateTime/Dutch/DurationParser.json

```diff
@@ -600,14 +600,14 @@
     ]
   },
   {
-    "Input": "Ik ben een kwartier uur weg",
+    "Input": "Ik ben een kwartier weg",
```

Comment:
Do we have a spec somewhere for the other form Mick mentioned? "Ik ben een kwart uur"

Reply:
added